### DISCUSSION TITLE
Android app gone from app store

### DISCUSSION THREAD

**Discussion by msymeonides** [rustdesk/rustdesk#5899] [General]
I saw the tweet about the Android app temporarily being taken down from the app store. Is there a timeline for putting it back up, or is there some way to sideload it without root access to the phone?

**Comment by discoveryX64** [comment]
An apk should be on the github page. If you're really desperate you can use the web client too. Was there ever an android app for rustdesk? I never saw.

**Reply by msymeonides** [reply]
Yeah it was on the Appstore and delisted: https://twitter.com/rustdesk/status/1707580722918150375

Which arch apk should I be downloading for a Samsung Galaxy S21? (sorry for ignorance)
